lua/src/macro_cache.cc

```diff
@@ -188,6 +257,34 @@ void macro_cache::_process_host(neb::host const& h) {
   _hosts[h.host_id] = h;
 }
 
+/**
+ *  Process a host group event.
+ *
+ *  @param hg  The event.
+ */
+void macro_cache::_process_host_group(neb::host_group const& hg) {
+  logging::debug(logging::medium)
+    << "macro_cache: process host group --- "
+    << "host group id = " << hg.id
+    << " ; name = " << hg.name;
+  _host_groups[hg.id] = hg;
```

Comment:
The *enabled* flag should be handled to check if group has to be added or removed from the cache.